wiki/Rules/en.md

```diff
@@ -1,77 +1,79 @@
+5.  **Keep your signatures and avatars reasonable.** If they're animated, they shouldn't strobe or flash excessively. They should also be of a filesize small enough for users with slower connections to not feel their presence on the page, ideally under half a megabyte (500 kilobytes) where possible.
+6.  **Don't post advertisements, referral links or any other annoying or intrusive content.** Things like affiliate and referral links are fun for the person posting them and an annoyance for everybody else, so don't do it. Autoloading or autoplaying content/embeds falls within this rule as well.
+7.  **Linking to applications or utilities of uncertain intent is not allowed.** Put simply, if there is any doubt as to the legality of said utilities, they are not welcome to be posted.
+
+Beatmap Submission Rules
 ---------------------------------
 
-These rules apply strictly to the unique nature of the [BSS](/wiki/Glossary) ([Beatmap](/wiki/Glossary) Submission System) threads. These threads are created via the osu! client when a beatmap is submitted through the [editor](/wiki/Beatmap_Editor).
+These rules apply strictly to beatmaps submitted via the in-game [Beatmap Submission System](/wiki/Beatmap_Submission_System). These threads/discussions are created via the osu! client when a beatmap is submitted through the [editor](/wiki/editor).
 
-1.  Do not create threads in the [Pending Submissions](/wiki/Beatmaps) and [WIP](/wiki/Glossary) boards. Use the appropriate subforum when creating a thread. (This obviously excludes BSS threads.)
-2.  Do not get off topic in BSS threads. BSS threads are about the parent beatmap. If you have an issue with something not directly pertaining to the beatmap, post it in the appropriate forum.
-3.  Do not steal others' beatmaps, presenting them as if they're mapped by you. Do not also upload or use other people's work without their explicit permission (including, but not limited to skins and guest difficulties).
+A set of more detailed rules for beatmap construction exists, known as the [Ranking Criteria](/wiki/Ranking_Criteria).
+
+1. **Make sure you have permission to use any content involved in your beatmap.** This includes songs, videos, hit sounds, graphics, and any other content that isn't your own creation. If you are unsure of where to find content that is free to use, we have a growing library of tracks available at the [Featured Artist](https://osu.ppy.sh/beatmaps/artists/) listing, all 100% licensed for use in _osu!_.
+2.  **Stay on topic in Beatmap discussions.** These threads/discussions are about the beatmap in question, and nothing else. If you have an issue with something not directly pertaining to the beatmap, post it in the appropriate forum.
+3.  **Do not plagiarise or attempt to steal the work of others.** Do not also upload or use other people's work without their explicit permission (including, but not limited to skins and guest difficulties).
```

Comment:
add a comma after `limited to`?
`including, but not limited to, skins and guest difficulties`